portfolio/src/main/webapp/index.html

```diff
@@ -2,82 +2,44 @@
-            final project for my class Software Design. I collaborate with two fellow classmates to design a web app for our
-            college, Olin College of Engineering.</li>
-          <li><a href="https://github.com/sd19spring/Death-Rate-Interactive-Map" target="blank">Interactive Map</a> - The 
-            interactive map shows the rates of death across the United States. When you hover over a state, a blurb will 
-            pop-up with the top ten leading causes of death in the United States and the rates for that particular state.</li>
-          <li><a href="https://popup.6seconds.org/" target="blank">Six Seconds POP-UP Festival</a> - As an intern, I used 
-            WordPress to build Six Seconds' website for United Nation's World Children's Day. The website is used all across the 
-            world to share the importance of emotional intelligence with children and encourage parents to raise their kids with
-            an emphasis on emotional awareness.</li>
-          <li><a href="https://www.blurb.com/bookstore/invited/8456952/5f42afbe2c71b68d858cb8a55f2fd29d0c587a2c" target="blank">
-            Photography Book</a> - In my Advanced Photography class, our semester long project was a photography book. My book is a 
-              reflection of my internal struggles with vulnerability and acceptance of who I am. My inspiration for my book was the 
-              desire to have stronger connections with others and feel at ease in my own skin, something I have been avoiding since
-              a young age.
-          </li>
-          <li><a href="http://poe.olin.edu/2019/followbot/" target="blank">FollowBot</a> - Designed and built a robot that
-            follows a person. The robots was primarily controlled by an Arduino. The computer vision used to follow a person
-            was run on a Raspberry Pi.
-          </li>
-      </ul>
-      <h2>Contact Me</h2>
-      <p>Email: <a href="mailto:kaoki@olin.edu">kaoki@olin.edu</a></p>
-      <p>Phone Number: <a href="tel:2102540921">(210)254-0921</a></p>
-      <p><a href="/documents/Resume.pdf" download>Resume</a></p>
-
+      <p>Welcome and hello to my portfolio! I hope you learn more about me and if you would like to more check out my contact info <a href="contact.html">
+          here</a>. Hope you have a great day!
+      
       <h2>Comments</h2>
       <h3>Previous Comments</h3>
-      <select id="language" onchange="requestTranslation();">
+      <!-- <select id="language" onchange="translateComment();">
             <option value="en" selected="selected">-- Select Your Language --</option>
             <option value="ar">Arabic</option>
             <option value="zh">Chinese</option>
+            <option value="en">English</option>
             <option value="fr">French</option>
             <option value="it">Italian</option>
             <option value="es">Spanish</option>
-      </select>
-
+      </select> -->
+      <!-- <ul id="results"></ul> -->
```

Comment:
Can probably just delete  rather than commenting. That's the beauty of source control, you can always go back if you need to :)